chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/presenter/CertifiedProductCsvPresenter.java

```diff
@@ -26,7 +26,7 @@
  * @author alarned
  *
  */
-public class CertifiedProductCsvPresenter implements CertifiedProductPresenter {
+public class CertifiedProductCsvPresenter implements CertifiedProductPresenter, AutoCloseable {
```

Comment:
Does AutoCloseable not need anything else added to it?

Reply:
The CertifiedProductPresenter interface already had a close method.  I removed it from that interface, and added the AutoCloseable interface to the class.  Basically, I just moved where the cloes() interface method was defined.